app/views/tasks/new.html.erb

```diff
@@ -9,5 +10,15 @@
     Hint: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form has a nice
     example of a simple HTML form!
   -->
+  <form action="" method="get" class="form-example">
+  <div class="form-example">
+    <label for="description">description </label>
+    <input type="text" name="description" id="description" required />
```

Comment:
Nothing in the tests thus far (parts 1 and 2) have dictated that you need the `required` attribute here.